config/environments/development.py

```diff
@@ -3,8 +3,10 @@
 from os import path
 
 from config.environments.default import Config
+from fsd_tech import configclass
 
 
+@configclass
```

Comment:
python Q - if our env config (dev.py, prod.py etc) inherit from default.py env (which already has the @configclass decorator on the base Config class), do we also need to apply this decorator to all sub classes ?